config/F16.config

```
@@ -93,6 +93,20 @@ process {
             }
         }
     }
+    withName: run_BedToIntervalList_picard {
```

```suggestion
    withName: run_BedToIntervalList_Picard {
```
Picard should be capitalized